docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -500,7 +500,8 @@
               If scale=true, than a deployment is triggered and the deployment is returned.
             body:
               application/json:
-                example: !include examples/app_tasks.json
+                type: instance.InstanceList | deploymentResult.DeploymentResult
+#                example: !include examples/app_tasks.json //todo fixit
```

Comment:
🤔 shouldn't the changes to `app_taks.json` fix the RAML generation errors. Why is this commented?

Reply:
forgot to uncomment this, thanks!